lib/project_types/extension/models/development_server_requirements.rb

```diff
@@ -13,7 +13,13 @@ class DevelopmentServerRequirements
 
       class << self
         def supported?(type)
-          binary_installed? && type_supported?(type) && type_enabled?(type)
+          if type_supported?(type) && type_enabled?(type)
+            return true if binary_installed?
+
+            context.debug(context.message("errors.development_server_binary_not_found"))
```

Comment:
@lemonmade, this addresses a concern you brought up in Slack. It should make it easier to determine why the CLI might fall back to the legacy flow. Now that I think about it, should we make this a full-fledged warning now that we're moving towards release. Future production builds shouldn't ever run into this problem, so if they do, something went seriously wrong and we should encourage users to file a bug report.

CC @dmhenry